tests/sles4sap/hana_install.pm

```diff
@@ -164,7 +127,7 @@ sub run {
         # If the ASSET_0 variable is defined, the test will attempt to download
         # the HANA media from the factory/other directory of the openQA server.
         record_info "Dowloading using ASSET_0";
-        download_hana_assets_from_server(target => $target, nettout => $tout);
+        $self->download_hana_assets_from_server(target => $target, nettout => $tout);
```

Comment:
was a bug present in the existing code? Is it related to newly added code in this PR?

Reply:
No, I have moved the sub from hana_install.pm to lib/sles4sap.pm as it is used on bone_install.pm as well.